config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :mechanics, only: [:show, :update]
+  resources :amusement_parks, only: [:show]
```

Comment:
SInce we are not updating a specific mechanic and instead creating a new relationship between a mechanic and a ride, you would not have a `:update` route for a mechanic here. Here is what your routes should look like:

```ruby
resources :mechanics, only: [:show] do
  resources :rides, only: [:create], controller: :mechanic_rides
end

resources :amusement_parks, only: [:show]
```